nixos/modules/services/networking/dae.nix

```diff
@@ -0,0 +1,42 @@
+{ config, pkgs, lib, ... }:
+
+{
+  options = {
+    services.dae = {
+      enable = lib.options.mkEnableOption (lib.mdDoc "the dae service");
+    };
+  };
+
+  config = lib.mkIf config.services.dae.enable {
+    environment.systemPackages = [ pkgs.dae ];
+
+    networking.firewall.allowedTCPPorts = [ 12345 ];
+    networking.firewall.allowedUDPPorts = [ 12345 ];
+
+    systemd.services.dae = {
+      unitConfig = {
+        Description = "dae Service";
+        Documentation = "https://github.com/daeuniverse/dae";
+        After = [ "network.target" "systemd-sysctl.service" ];
+        Wants = [ "network.target" ];
+      };
+
+      serviceConfig = {
+        User = "root";
+        ExecStartPre = "${lib.getExe pkgs.dae} validate -c /etc/dae/config.dae";
+        ExecStart = "${
+            lib.getExe pkgs.dae
+          } run --disable-timestamp -c /etc/dae/config.dae";
```

Comment:
```suggestion
        ExecStart = "${lib.getExe pkgs.dae} run --disable-timestamp -c /etc/dae/config.dae";
```

Reply:
How about doing `ExecStart = "${lib.getExe cfp.package} run --disable-timestamp -c /etc/dae/config.dae";`

Just like it's done in [here](https://github.com/NixOS/nixpkgs/blob/aad0e18f8d86daf57cfb30e13958f70b6a73705c/nixos/modules/services/web-apps/guacamole-server.nix#L75). You will need to add the `package` config attribute, like in [here](https://github.com/NixOS/nixpkgs/blob/aad0e18f8d86daf57cfb30e13958f70b6a73705c/nixos/modules/services/web-apps/guacamole-server.nix#L13).

Reply:
Can you please add the missing cfg.package, like it's shown in the guacamole module ? It's a one line change almost.

Reply:
> Can you please add the missing cfg.package, like it's shown in the guacamole module ? It's a one line change almost.

Thanks for suggestion! Already included in latest commit :)